test/smoke/framework/cli.go

```diff
@@ -16,9 +16,23 @@ package framework
 
 import (
 	"fmt"
+	"os"
 	"os/exec"
 )
 
+// CheckCliBinaryExist checks if the CLI binary does exist
+func CheckCliBinaryExist() (bool, error) {
+	path, err := getEnvOperatorCliPath()
+	if err != nil {
+		return false, err
+	}
+
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return false, nil
```

Comment:
Please cover also branch when err is not nil but also not IsNotExist()

Reply:
updated